### PR TITLE
Create Vue 3 blog post

### PR DESCRIPTION
**Pull request by ferferga**
Every release has been made on Jellyfin Vue repo, this is ready to be merged as soon as possible

### CODE REVIEW

**Review by Shadowghost — APPROVED** [review]

File: blog/2023/04-03-vue_vue3/index.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+That include the majority of TVs, sadly. **Chrome/Edge/Firefox/Safari, both in desktop and mobile platforms are still fully compatible**.
+If the web browser of your device still gets updates, it'll likely have no issues running Jellyfin Vue.
+
+Although the affected userbase will probably be small, I'm sorry for those who might be affected by this decision but,
+given our limited resources, looking forward is our only way to go if we want to provide the best alternative web client for Jellyfin.
+You can still run the pre-Vue3 version (go to *Downloads & changelogs* to know more), although it will not receive any support from us.
+
+### A note on privacy 🕵🏻
+
+[I wrote an small disclaimer explaining how Jellyfin Vue (and jellyfin Web) works](https://github.com/jellyfin/jellyfin-vue#privacy-disclaimer-),
```
Comment (Shadowghost) [inline blog/2023/04-03-vue_vue3/index.mdx:39]:
```suggestion
[I wrote an small disclaimer explaining how Jellyfin Vue (and Jellyfin Web) works](https://github.com/jellyfin/jellyfin-vue#privacy-disclaimer-),
```

File: blog/2023/04-03-vue_vue3/index.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+* [v0.2.0](https://github.com/jellyfin/jellyfin-vue/releases/tag/0.2.0): This is the last release powered by Vue 2.
+* [v0.3.0](https://github.com/jellyfin/jellyfin-vue/releases/tag/0.3.0): This is the first release powered by Vue 3. This was tagged right after we merged the branch we've been working on these past months.
+* [v0.3.1](https://github.com/jellyfin/jellyfin-vue/releases/tag/0.3.1): Same as v0.3.0, but after all the pending dependency updates have been merged.
+
+---
+
+# What's Jellyfin Vue?
+
+Jellyfin Vue is an alternative web client some of us started working on 2020. Jellyfin Web had a lot of maintenability
```
Comment (Shadowghost) [inline blog/2023/04-03-vue_vue3/index.mdx:72]:
```suggestion
Jellyfin Vue is an alternative web client some of us started working on in 2020. Jellyfin Web had a lot of maintenability
```

**Review by Shadowghost — APPROVED** [review]

File: blog/2023/04-03-vue_vue3/index.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+It even allowed us to play with SSR. However, it forced a really rigid structure on us, besides
+[SSR turned out to be a burden](https://github.com/jellyfin/jellyfin-vue/discussions/1620).
+We had even to use some hacks to avoid killing playback while transitioning between pages.
+
+As part of our Vue 3 migration, we switched to an standard Vue 3 project structure
+(Vue 3, Vite, TypeScript and, in the future, Vitest), which allowed us to remove many of the hacks
+and legacy code we used to overcome Vue and Nuxt 2 limitations. Beside that, Vue 3
+only supports ES6-compatible browsers. We also use [some syntax](https://github.com/jellyfin/jellyfin-vue/blob/master/frontend/src/main.ts#L2)
+that's only compatible with ES2022. All of this means that, as part of this upgrade, **we also deprecated all legacy support for outdated browsers**.
+That include the majority of TVs, sadly. **Chrome/Edge/Firefox/Safari, both in desktop and mobile platforms are still fully compatible**.
```
Comment (ThibaultNocchi) [inline blog/2023/04-03-vue_vue3/index.mdx:30]:
```suggestion
That includes the majority of TVs. **Chrome/Edge/Firefox/Safari, both in desktop and mobile platforms are still fully compatible**.
```

I'm not sure if we need to say "sadly" :p

**Review by ThibaultNocchi — COMMENTED** [review]
Great. What I'd add, if you have some time, is a small part about your future UX plan. That may spark interest.

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**